extensions/oidc/runtime/src/test/java/io/quarkus/oidc/runtime/OidcTenantConfigImpl.java

```diff
@@ -0,0 +1,1238 @@
+package io.quarkus.oidc.runtime;
+
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.configuration.TrimmedStringConverter;
+import io.smallrye.config.WithConverter;
+
+final class OidcTenantConfigImpl implements OidcTenantConfig {
```

Comment:
@michalvavrik Can you clarify please what does this class do ?

Reply:
It is a test class that I use to test that mapper between a `@ConfigMapping` interface and former `@ConfigGroup` class maps every single interface method. If I do:

```
configGroupClass.nameOne = interfaceGroup.nameOne();
```
then when `nameOne` is invoked, at least I know that it wasn't forgotten.

Reply:
Oh sorry, it is a test class